src/app/Repositories/CoronaStatusRepository.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+
+namespace App\Repositories;
+
+
+use App\Models\CoronaStatus;
+
+class CoronaStatusRepository
+{
+    public function findByCode(string $code): ?CoronaStatus
+    {
+        return CoronaStatus::where('code', '=', $code)
+            ->first();
```

Comment:
```suggestion
        return CoronaStatus::query()
            ->where('code', '=', $code)
            ->first();
```
arba
```suggestion
        return CoronaStatus::whereCode($code)
            ->first();
```
Abu variantai veiks po to, kai įdiegsi `barryvdh/laravel-ide-helper` ir sukursi modeliams `php doc`. Rekomenduoju lengvoms užklausoms naudoti antrą variantą su `CoronaStatus::whereCode($code)`, o sudėtingoms pirmąjį su `CoronaStatus::query()`, kadangi `whereSomething` gali panaudoti tik pačioje pradžioje, o toliau vis tiek teks naudoti standartinį `where` ir kodas atrodys padrikai.